### PR TITLE
Cleans up some old commands and adds some missing subcommands

### PR DESCRIPTION
**Pull request by SirMonkeyBoy**

### CODE REVIEW

File: docs/commands.md [hunk 1]
```diff
@@ -150,8 +141,6 @@
     * Shows you your mcmmo stats
   * /tags
     * Opens the tags GUI to select a tag
-  * /titan pack
```
Comment (JL-III) [inline docs/commands.md:153]: `/titan pack` still exists, can you re-add this?

Comment (SirMonkeyBoy) [reply docs/commands.md:153]: image
It doesn't show up for me

Comment (JL-III) [reply docs/commands.md:153]: okay, this is fixed. can you re-add this now.

Comment (SirMonkeyBoy) [reply docs/commands.md]: its now added back should be able to merge unless any other things need a change

**Pull request merged**